Sources/Testing/Traits/Trait.swift

```diff
@@ -92,41 +97,40 @@ public protocol Trait: Sendable {
   /// If this trait's type does not conform to ``TestScoping`` and its
   /// associated ``Trait/TestScopeProvider`` type is the default `Never`, then
   /// this method returns `nil` by default. This means that instances of this
-  /// trait will not provide a custom scope for tests to which they're applied.
+  /// trait don't provide a custom scope for tests to which they're applied.
   func scopeProvider(for test: Test, testCase: Test.Case?) -> TestScopeProvider?
 }
 
-/// A protocol that allows providing a custom execution scope for a test
-/// function (and each of its cases) or a test suite by performing custom code
-/// before or after it runs.
+/// A protocol that tells the test runner to run custom code
+/// before or after it runs a test suite or test function.
 ///
-/// Types conforming to this protocol may be used in conjunction with a
+/// Use types that conform to this protocol in conjunction with a
 /// ``Trait``-conforming type by implementing the
-/// ``Trait/scopeProvider(for:testCase:)-cjmg`` method, allowing custom traits
-/// to provide custom scope for tests. Consolidating common set-up and tear-down
-/// logic for tests which have similar needs allows each test function to be
-/// more succinct with less repetitive boilerplate so it can focus on what makes
-/// it unique.
+/// ``Trait/scopeProvider(for:testCase:)-cjmg`` method, so your custom traits
```

Comment:
Awkward phrasing: "Use types [...], so your custom traits [...]". Can we revise?

Reply:
OK I've had a go at this, see what you think.

Reply:
Looks good, thanks!